src/jabs_postprocess/cli/utils.py

```diff
@@ -0,0 +1,75 @@
+import json
+from pathlib import Path
+from typing import Dict, Any, List
+
+
+def load_json(source: Path | str | None) -> Dict[str, Any] | List[Any] | None:
+    """Load JSON from a file path, JSON string, or return None.
+
+    Args:
+        source: Can be:
+            - Path object pointing to a JSON file
+            - String containing a file path to a JSON file
+            - String containing raw JSON content
+            - None (returns None)
+
+    Returns:
+        Parsed JSON (dict, list, or other JSON-serializable type), or None if input is None
+
+    Raises:
+        ValueError: If source cannot be parsed as JSON or loaded from file
+        TypeError: If source is not a supported type
+    """
+    if source is None:
+        return None
+
+    if isinstance(source, Path):
+        # It's a Path object, read from file
+        try:
+            with open(source, "r", encoding="utf-8") as f:
+                return json.load(f)
+        except FileNotFoundError:
+            raise ValueError(f"File not found: {source}")
+        except json.JSONDecodeError as e:
+            raise ValueError(f"Invalid JSON in file {source}: {e}")
+
+    elif isinstance(source, str):
```

Comment:
maybe the `isinstance(source, str)` block could be simplified by checking Path(source).exists() first to see if it is a path, then fall back to parsing the string as JSON and if that fails finally check if it is one of the quoted strings?

I think this could avoid the multiple attempts to open it as a file